packages/block-library/src/post-navigation-link/index.php

```diff
@@ -28,14 +28,9 @@ function render_block_core_post_navigation_link( $attributes, $content ) {
 	if ( isset( $attributes['textAlign'] ) ) {
 		$classes .= " has-text-align-{$attributes['textAlign']}";
 	}
-	$styles = '';
-	if ( isset( $attributes['style']['typography']['writingMode'] ) ) {
-		$styles = "writing-mode: {$attributes['style']['typography']['writingMode']};";
-	}
 	$wrapper_attributes = get_block_wrapper_attributes(
 		array(
 			'class' => $classes,
-			'style' => $styles,
 		)
 	);
```

Comment:
Since this change will be bundled in with the packages update for core, would it be worth trying to land the change to core's `safe_style_css` list before attempting to land this Gutenberg PR?

Reply:
I see, I think that's certainly true. 

Would it be ideal to wait to merge this PR until a patch has been submitted to the core that adds `writing-mode` to the `safe_style_css` list and is committed in the core?

Reply:
I don't think it necessarily needs to be committed to core before merging this PR, but I'd recommend opening a trac ticket and corresponding PR for `wordpress-develop` before merging if you have the time, that way the change will be ready to land in time.